masonry_core/src/passes/paint.rs

```diff
@@ -58,16 +63,17 @@ fn paint_widget(
     state.item.request_paint = false;
     state.item.needs_paint = false;
 
-    let clip = state.item.clip_path;
-    let has_clip = clip.is_some();
-    let transform = state.item.window_transform;
-    let scene = scenes.get(&id).unwrap();
+    let has_clip = state.item.clip_path.is_some();
+    if !is_stashed {
```

Comment:
I don't know ... it just feels like if the widget its stashed, maybe it shouldn't even be getting this far and wanting to do all of this work.